tests/test_fall_back.py

```diff
@@ -1,40 +1,89 @@
-# from typing import Dict, Tuple
-# from skyplane import compute
-# from skyplane.api.config import TransferConfig
-# from skyplane.planner.planner import MulticastDirectPlanner, Planner
-# from skyplane.utils.fn import do_parallel
+from skyplane.api.config import TransferConfig
+from skyplane.cli.cli_transfer import SkyplaneCLI
+from skyplane.config import SkyplaneConfig
+from skyplane.planner.planner import MulticastDirectPlanner
 
-# quota_limits = {
-#     "aws": [
-#         {"on_demand_standard_vcpus": 5, "spot_standard_vcpus": 5, "region_name": "us-east-1"},
-#     ],
-#     "gcp": {
-#         "us-east1": 8,
-#     },
-#     "azure": {
-#         "uaenorth": 4,
-#     },
-# }
 
+def test_fall_back_multiple_limits_and_types():
+    # Regions for each provider
+    regions = {
+        "aws": "us-east-1",
+        "azure": "uaenorth",
+        "gcp": "us-east1",
+    }
 
-# def test_fall_back():
-#     transfer_config = TransferConfig()
-#     planner = MulticastDirectPlanner(n_instances=8, n_connections=100, transfer_config=transfer_config)
-#     planner.quota_limits = quota_limits
+    # Different quota limits
+    tests = [
```

Comment:
created three fake quota limits

Reply:
I moved the fall_back test to the tests/no_cloud directory since every test inside that gets run automatically during unit tests